parts/configure-swarm-cluster.sh

```diff
@@ -161,16 +163,13 @@ echo "Installing and configuring docker"
 
 installDocker()
 {
-  for i in {1..10}; do
-    wget --tries 4 --retry-connrefused --waitretry=15 -qO- https://get.docker.com | sh
-    if [ $? -eq 0 ]
-    then
-      # hostname has been found continue
-      echo "Docker installed successfully"
-      break
-    fi
-    sleep 10
-  done
+  apt-get update

```

Comment:
all "apt" commands should have retries around them.  Please see other areas of code where we do this.